freqtrade/constants.py

```diff
@@ -22,6 +22,12 @@
 AVAILABLE_PAIRLISTS = ['StaticPairList', 'VolumePairList']
 DRY_RUN_WALLET = 999.9
 
+# Urls to exchange markets, insert quote and base with .format()
+_EXCHANGE_URLS = {
```

Comment:
Please remove these completely - we used to include a link to the pair in the RPC messages, but that has been removed (apparently I forgot to remove the url's ...)

Reply:
I think we were discussing this in your Binance refactoring and we decided to keep them back then, in case we'd still need them for something.. but yea, I'm fine with removing completely. Easy enough to add back should we need them at any point.